mypy.ini

```diff
@@ -14,14 +14,11 @@ disallow_any_generics = True
 warn_no_return = True
 no_implicit_optional = True
 warn_redundant_casts = True
+warn_unused_ignores = True
 
 # Display the codes needed for # type: ignore[code] annotations.
 show_error_codes = True
 
-# It's useful to try this occasionally, and keep it clean; but when
-# someone fixes a type error we don't want to add a burden for them.
-#warn_unused_ignores = True
```

Comment:
I think the reason we originally disabled this was because some ignores were necessary only on Python 2 or Python 3, but not both due to disagreements in typeshed where one was clearly wrong; that's no longer necessary.